Cargo.toml

```diff
@@ -30,6 +30,7 @@ members = [
     "node/router",
     "node/store",
     "node/tcp",
+    "metrics",
```

Comment:
I'm wondering if this shouldn't be `node/metrics` :thinking:

Reply:
done